l10n_it_fatturapa/demo/account_invoice_fatturapa.xml

```diff
@@ -490,5 +490,18 @@
             <field name="country_id" ref="base.ae" />
             <field name="email">info@mycompany.it</field>
         </record>
+
+        <record id="res_partner_fatturapa_5" model="res.partner">
```

Comment:
Puoi mettere queste modifiche in un commit dedicato? Di solito è meglio modificare solo un modulo in ogni commit per vari motivi, ad esempio poi è più facile fare il revert delle modifiche.
Rif https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message

Reply:
Fatto.